spec/workers/sync_users_job_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+describe SyncUsersJob do
+  describe "#perform_async" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.